onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -23,24 +23,23 @@ class InferenceSessionGetGraphWrapper : public InferenceSession {
                                            const Environment& env) : InferenceSession(session_options, env) {
   }
 
-  const Graph& GetGraph() {
+  const Graph& GetGraph() const {
     return model_->MainGraph();
   }
 
-  const SessionState& GetSessionState() {
+  const SessionState& GetSessionState() const {
     return InferenceSession::GetSessionState();
   }
 };
 
 namespace test {
 
-#if !defined(ORT_MINIMAL_BUILD)
 // Same Tensor from ONNX and ORT format will have different binary representation, need to compare value by value
```

Comment:
Are you using this compare functions in a minimal build? If not you get a build error about removing an unused file scope function.